### PR TITLE
[WIP] Democracy guidebook

### PR DESCRIPTION
**Pull request by MyronFanQiu**

### CODE REVIEW

File: website/translated_docs/zh-CN/democracy-guide.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+![voting_detail](assets/democracy/voting_detail.png)
+
+5. 进入执行阶段
+在每个提案表决时间结束时，通过表决的议案会进入执行阶段。根据提案类型的不同，执行分为立即执行和延后执行。本文前述的默认普通提案会延后特定时间执行。可以在如下页面查看具体执行时间。
+![scheduled](assets/democracy/scheduled.png)
+
+## 规则参数解析及注意事项
+
+1. 图像创建必须是能够被Root权限执行的交易，例如staking.SetValidatorCount或者market.setPunishment。否则即使进入执行阶段，最终也会执行失败。
+2. 提交议案和复议都会要求抵押一定数值的CRU。预览网上设置的金额大小为**10CRU**。提案一旦进入表决阶段，锁定的金额会自动解锁。但是需要主要的是，当有若干提案候选时，一旦复议某提案，除非该提案进入表决阶段，锁定的CRU是无法解锁的。预览网上，**每7天**会选择得到复议最高的提案进入表决阶段。
```
Comment (yyd106) [inline website/translated_docs/zh-CN/democracy-guide.md:48]: 注意

**Review by yyd106 — APPROVED** [review]

**Review by yyd106 — APPROVED** [review]

**Comment by yyd106** [comment]
Good to go

File: docs/democracy-guide.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+id: democracyGuide
+title: Democracy Guide
+sidebar_label: Democracy Guide
+---
+
+The Democracy module is an important part of Crust Network, reflecting the advantages of Crust on-chain governance and degree of decentralization. Since the design of the Democracy module is modelled after the British parliamentary system that by nature contains many aspects, this tutorial mainly focuses on the main processes, specific practices, rules, and parameters of the Democracy module for the ease of reading.
+
+
+
+## Process of Democracy module
```
Comment (zikunfan) [inline docs/democracy-guide.md:11]: Add ToC of democracy: proposals&referenda, tech commitee...

File: docs/democracy-guide.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+### Main processes
```
Comment (zikunfan) [inline docs/democracy-guide.md:13]: Change this main process to proposal process

Comment (zikunfan) [inline docs/democracy-guide.md:13]: Add section explaination of this process(referenda/proposal process)

**Review by zikunfan — CHANGES_REQUESTED** [review]

**Review by zikunfan — APPROVED** [review]
lgtm

File: docs/democracy-guide.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
+---
+id: democracyGuide
+title: Democracy Guide
+sidebar_label: Democracy Guide
+---
+
+The Democracy module is an important part of Crust Network, reflecting the advantages of Crust on-chain governance and degree of decentralization. The whole picture of Democracy module contains the following parts:
+
+1. [**Proposals&Referenda**](#1-proposalsreferenda): General democracy module that every one can submit a proposal and vote for the proposal to make Crust Network better.
```
Comment (yyd106) [inline docs/democracy-guide.md:9]: every one -> everyone

**Pull request merged**